examples/validationTransformer/README.md

```diff
@@ -1,8 +1,8 @@
 # a transformer plugin performing validation
 
-[base]: ../../docs/glossary.md#base
+[base]: https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#base
 [kubeval]: https://github.com/instrumenta/kubeval
-[plugin]: ../../docs/plugins
+[plugin]: ../../plugins
```

Comment:
Maybe the right link is here https://github.com/kubernetes-sigs/kustomize/tree/master/plugin

Reply:
@koba1t  I will put this link and update